rust/htp/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
-name = "htp"
+name = "suricata-htp"
 authors = ["ivanr = Ivan Ristic <ivanr@webkreator.com>", "cccs = Canadian Centre for Cyber Security"]
```

Comment:
should we add @catenacyber here?

Reply:
I think its more about who to contact in case of issue.. But hmm.. This field is deprecated, I don't think I got a warning about it though:

https://doc.rust-lang.org/cargo/reference/manifest.html#the-authors-field